www/includes/utility.php

```diff
@@ -893,6 +879,8 @@ function entities_to_numbers($string) {
 
 function make_member_url($name, $const = '', $house = HOUSE_TYPE_COMMONS, $pid = NULL) {
 
+    # convert name to ISO so we have sensible URLs
+    $name = iconv('UTF-8', 'ISO-8859-1//TRANSLIT', $name);
```

Comment:
You will presumably need to `urlencode` the name if you're keeping the accents in the URL? You could replace all the hex in `s2` with the corresponding UTF-8 hex and then it would still ASCIIfy them all? Or do you mean you did that and that was what didn't work.